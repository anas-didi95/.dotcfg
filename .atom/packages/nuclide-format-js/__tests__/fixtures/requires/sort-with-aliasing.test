const Y = require('AB');
const Z = require('A').B.c.Y;

const x = Y + Z;
