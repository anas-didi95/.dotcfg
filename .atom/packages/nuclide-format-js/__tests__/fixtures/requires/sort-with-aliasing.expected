const Z = require('A').B.c.Y;
const Y = require('AB');

const x = Y + Z;
